src/RpcServer.h

```diff
@@ -9,6 +9,7 @@
 #define NT_RPCSERVER_H_
 
 #include <atomic>
+#include <chrono>
```

Comment:
These chronos where an initial version where the functions took a std::milliseconds value. I'll fix them.